### ISSUE TITLE
Consume configurable lab result input in write_lab_results.R

### ISSUE THREAD

**Issue by pbchase** [ctsit/tt_engine#156]
To consume clinical lab results in `write_lab_results.R`, we will need to read a different lab results file produced on a different schedule from the exist lab results file. Similarly, we will need to write rejected lab records to a different file. The new input file will have the same format as the exist input file. The new rejects file should also have the same format as the old rejects file. 

To implement this, we will need to make a few changes: 

1. Modify https://github.com/ctsit/tt_engine/blob/7e5462163bb9ee945c6eac12a898376b06ccb7b4/R/data_fetching.R#L130 to accept a pattern as a parameter. We should replace the call to `get_results_data.sh` with `get_latest_file.sh` and pass the parameter to it. As we do this we can delete `get_results_data.sh` as it is used no where else.
1. Modify `write_lab_results.R` at https://github.com/ctsit/tt_engine/blob/7e5462163bb9ee945c6eac12a898376b06ccb7b4/write_lab_results.R#L20 to provide the new parameter as a variable. 
1. Modify `write_lab_results.R` at https://github.com/ctsit/tt_engine/blob/7e5462163bb9ee945c6eac12a898376b06ccb7b4/write_lab_results.R#L47 and at https://github.com/ctsit/tt_engine/blob/7e5462163bb9ee945c6eac12a898376b06ccb7b4/write_lab_results.R#L58 to set the two reject file names via variables. 

We will also need to set the three variables with the three filenames. I would not recommend moving these to configuration. I think they should be embedded in the code much as they are today. The correct set of file names should probably be selected via an argument with the default behavior using the existing set of file names. 

How the two sets of file names should existing the code is up to the developers discretion. Both sets of filenames could be written into a small dataframe with the correct row selected by a filter. 

The existing file name path and basename are:
```
input - "data_out/epic/EPIC_TO_REDCAP_"
epic_reject - "data_out/redcap/redcap_to_epic_rejected_lab_data_"
wh_reject - "data_out/redcap/redcap_to_wh_rejected_lab_data_"
```

The new file name patterns should be:
```
input - "data_out/redcap/redcap_to_idr_clinical_result_request_"
epic_reject - "data_out/redcap/redcap_to_idr_rejected_lab_data_"
wh_reject - "data_out/redcap/redcap_to_wh_rejected_lab_data_"
```

It's unclear if we should change `wh_reject` in the list of new patterns. Let's go with the existing pattern for now.

**Comment by pbchase** [comment]
Matt send inputs for this today.

**Comment by ljwoodley** [comment]
Yep, I'll get started with this issue tomorrow.

**Comment by pbchase** [comment]
Addressed by PR #162
